guides/source/caching_with_rails.md

```diff
@@ -512,12 +512,21 @@ class ProductsController < ApplicationController
 end
 ```
 
-### A note on weak ETags
+### Strong v/s Weak ETags
+Weak ETags allow semantically equivalent responses to have the same ETags, even
+if their bodies do not match exactly. This is useful when we don't want the page
+to be regenerated for minor changes in response body.
 
-Etags generated by Rails are weak by default. Weak etags allow semantically equivalent responses to have the same etags, even if their bodies do not match exactly. This is useful when we don't want the page to be regenerated for minor changes in response body. If you absolutely need to generate a strong etag, it can be assigned to the header directly.
+```ruby
+  response.weak_etag = response.body # => W/"618bbc92e2d35ea1945008b42799b0e7" → weak ETag
+```
```

Comment:
No need to mention `response.weak_etag=` as Rails will always generate weak ETags by default.